app/controllers/concerns/classified_listings_toolkit.rb

```diff
@@ -13,11 +13,14 @@ def publish_listing
     @classified_listing.save
   end
 
+  # TODO: why not just @classified_listing.update(listing_params)?
```

Comment:
no, but you could use `assign_attributes` not to trigger an update at this stage I guess

Reply:
ps, don't forget to add the name of the team, it's clearer for onlookers :)

something like `# TODO: [thepracticaldev/oss]`

Reply:
I had `assign_attributes` locally for a while, it seemed to work fine. I reverted it before pushing though, because this is idiosyncratic enought that I figured there must be context. Anyway, I'll jut change it back.

Re the TODO: I can change it, but given that I wanted to start on the next PR within a few days, the comments should be around too long anyway.

Reply:
My guess would be that some of these params might come through as nil or blank when not changed and this prevents them from being incorrectly set. That is totally just a guess. You could still refactor though with something like
```ruby
listing_params.keep_if{|k, v| v.present?}
@classified_listing.assign_attributes(listing_params)
```

Reply:
@mstruve Was thinking the same, but the way it's currently implemented only protects against `nil` and not blank, so I was a bit reluctant to change it before at least double-checking if someone knows more. 

Anyway, I've decided to leave this method as-is for now and keep the changes for the 3rd PR.